pkg/products/threescale/prometheusRules.go

```diff
@@ -42,6 +43,13 @@ func (r *Reconciler) newAlertReconciler(logger l.Logger, installType string, ctx
 		operatorAlertNamePrefix = "3scale-operator-"
 	}
 
+	// conditional alert level for 3scale UI portals
+	threescaleAlertLevel := "critical"
+	customDomainEnabled, _ := strconv.ParseBool(r.Config.GetCustomDomainEnabled()) // value is false in case of error
+	if customDomainEnabled {
+		threescaleAlertLevel = "warning"
+	}
```

Comment:
If this is removed the alert level won't be conditional anymore. The existing *UIBBT alerts need to be `critical` for normal clusters and `warning` for custom domain enabled clusters, right?

Reply:
I need to check this, but I think for now all the *UIBBT alerts will be warnings for both installation setups. This is because of the existing cluster that have custom domains currently can not be detected. Once we can detect these snowflake clusters then doing this installation specific alerts levels will be needed.